src/app/modules/shared/pipes/filter-project/filter-project.pipe.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { FilterProjectPipe } from './filter-project.pipe';
+
+describe('FilterProjectPipe', () => {
+  it('create an instance', () => {
+    const pipe = new FilterProjectPipe();
+    expect(pipe).toBeTruthy();
+  });
+
+  it('test method of pipe', () => {
+    expect(new FilterProjectPipe().transform([], '')).toEqual([]);
```

Comment:
improve the test, you can create a mock with projects data and test the transform method.